code/modules/mob/living/living_fov.dm

```diff
@@ -57,6 +57,8 @@
 		if(fov_type > highest_fov)
 			highest_fov = fov_type
 	fov_view = highest_fov
+	if(HAS_TRAIT_FROM(src, TRAIT_UNDENSE, LEANING_TRAIT))
+		fov_view += 30
```

Comment:
might be better to just make a new trait for this, this code doesn't make much sense out of context

Reply:
👍